mobile/components/ReviewCitizenFormSheet.tsx

```diff
@@ -347,12 +369,14 @@ export default function ReviewCitizenFormSheet({
     >
       <Sheet.Overlay />
       <Sheet.Frame>
-        {isUploading ? (
+        {isUploading && attachments ? (
           <YStack flex={1} justifyContent="center" alignItems="center">
             <MediaLoading
               progress={uploadProgress}
               isUploading={isUploading}
+              uploadedAttachments={Object.entries(attachments).map(([, attachments]) => attachments.filter((a) => a.uploaded)).flat().length}
               onAbortUpload={onAbortUpload}
+              confirmAbort
```

Comment:
Avem cazuri in care nu afisam abort confirmation? Gen pe questionId or something?